src/modules/Auth/components/ButtonMenu.tsx

```diff
@@ -0,0 +1,95 @@
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import Logout from '@mui/icons-material/Logout';
+import Settings from '@mui/icons-material/Settings';
+import Box from '@mui/material/Box';
+import IconButton from '@mui/material/IconButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import Tooltip from '@mui/material/Tooltip';
+import { makeStyles } from '@mui/styles';
+import * as React from 'react';
+
+const useStyles = makeStyles(() => ({
+  logOut: {
+    color: '#FF0000',
+  },
+}));
+
+export const ButtonMenu = () => {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  return (
+    <>
+      <Box sx={{ display: 'flex', alignItems: 'center', textAlign: 'center' }}>
```

Comment:
You can pass this styles as a properties:
```suggestion
      <Box dispaly="flex" alignItems="center" textAlign="center">
```
I prefer to use properties instead `sx`. But your solution works too and you don't have to change it according to my proposal

Reply:
I leave it like this